openflow_input/standard-1.5

```diff
@@ -830,26 +830,55 @@ enum ofp_stats_reply_flags(wire_type=uint16_t, bitmask=True) {
     OFPSF_REPLY_MORE = 0x1,
 };
 
-/* Multipart Message Types */
+/* 
+ * Multipart Message Types 
+ *
+ * OF1.5 renames the OFPMP/ST for:
+ * flow (1) -> flow_stats (17)
+ * aggregate (2) -> aggregate_stats (2)
+ * table (3) -> table_stats (3)
+ * group (6) -> group_stats (6)
+ * meter (9) -> meter_stats (9)
+ * meter_config (10) -> meter_desc (10)
+ * 
+ * OF1.4 renamed:
+ * queue (5) -> queue_stats (5)
+ * which was retained as "queue" in standard-1.4.
+ *
+ * Using different names for the same message
+ * in Loxi causes duplicate enum entries that
+ * map to essentially the same underlying message.
+ * For simpliciy, we can retain the original, 
+ * OF1.3-, naming convention to reduce complexity.
+ * This is why OF1.4+ ofp_stats_type enums from 
+ * OF1.3- remain as OFPST_* and do not appear as
+ * OFPMP_*.
+ *
+ * Using the _STATS is certainly more explicit; 
+ * however, that would perhaps necessitate changing
+ * OF1.3- OFPMP/ST names, which would break existing
+ * applications that use OF1.3- OFPMP/ST names,
+ * such as Floodlight, ONOS, etc.
+ */
 enum ofp_stats_type(wire_type=uint16_t) {
     OFPST_DESC = 0,
     OFPST_FLOW_DESC = 1,
```

Comment:
I see... after a look at the OF1.5.1 I am less confused.

I am not quite sure how to best deal with the renaming of the flow_stats_request and the introduction of the 'lightweight' stats request. On the one hand the rename will break code, but then again the code will need special case handling anyway due to the introduction of the OXS stuff.

Overall I would maybe tend to continue to call the  `stats_type 1` message `flow_stats_request/response`, because it seems the closest to what the old message was useful for. But I guess a case could be made that `_desc` is more consistent with other parts of the API.

It might not matter in practice for us, because we might not switch to 1.5... 

@rlane - any opinions?